pyproject.toml

```diff
@@ -34,7 +36,8 @@ docs = [
 ]
 test = [
     'pytest',
-    'asdf-astropy >= 0.2.0'
+    #'asdf-astropy >= 0.2.0'
+    'asdf-astropy @ git+https://github.com/braingram/asdf-astropy.git@split_core'
```

Comment:
To be removed prior to merge. This is at the moment required due to examples using tags new to this PR which are not yet supported in asdf-astropy (support added in: https://github.com/astropy/asdf-astropy/pull/219).